kotlin-mui/src/jsMain/generated/mui/base/useSlider.types.kt

```diff
@@ -194,6 +194,11 @@ external interface UseSliderReturnValue {
      */
     var range: Boolean
 
+    /**
+     * Ref to the root slot's DOM node.
+     */
+    var rootRef: react.RefCallback<web.html.HTMLElement>?
```

Comment:
`import web.html.HTML`?

Reply:
Will become `web.dom.Element` in the next update